packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js

```diff
@@ -0,0 +1,50 @@
+export default class IpByoipAggregateController {
+  /* @ngInject */
+  constructor($scope, IpByoipService) {
+    this.$scope = $scope;
+    this.IpByoipService = IpByoipService;
+    this.ip = $scope.currentActionData.ipBlock;
+    this.aggregationIps = [];
+    this.selectedAggregationIp = {};
+    this.isLoaded = false;
+    this.errorMessage = null;
+
+    IpByoipService.getAvailableAggregationConfigurations(this.ip.ipBlock)
+      .then((response) => {
+        this.aggregationIps = response.data;
+        if (this.aggregationIps.length) {
+          this.selectedAggregationIp = this.aggregationIps[0];
+        }
+      })
+      .catch((error) => {
+        this.errorMessage = error.data.message;
+      })
+      .finally(() => {
+        this.isLoaded = true;
+      });
```

Comment:
This should not be done in the component routing that you should set up as requested in https://github.com/ovh/manager/pull/10592#discussion_r1450133019

Reply:
Mark as resolved (see: https://github.com/ovh/manager/pull/10592#discussion_r1450694853)